packages/changed-elements-react/src/api/ElementQueries.ts

```diff
@@ -128,7 +128,7 @@ export const queryEntryDataBulk = async (
   iModel: IModelConnection,
   elementIds: string[],
   chunkSize = 1000,
-  updateFunc?: () => void,
+  updateFunc?: (pct: number) => void,
```

Comment:
Also adddocs on expectations of when this is called
```suggestion
  updateFunc?: (percentage: number) => void,
```

Reply:
addressed, also modified to reflect chunksize processed in each iteration